webextension/js/popup.js

```diff
@@ -144,7 +133,11 @@ const Logic = {
     browser.browserAction.setBadgeBackgroundColor({color: ""});
     browser.browserAction.setBadgeText({text: ""});
     storage.browserActionBadgesClicked.push(extensionInfo.version);
-    browser.storage.local.set({browserActionBadgesClicked: storage.browserActionBadgesClicked});
+    // use set and spread to create a unique array
+    const browserActionBadgesClicked = [...new Set(storage.browserActionBadgesClicked)];
+    browser.storage.local.set({
+      browserActionBadgesClicked
+    });
```

Comment:
You and your clever JS applications. Thanks for the comment - would never have understood the reasoning behind this. 😜